apps/integration/integration/automation/runtests.sh

```diff
@@ -33,20 +55,73 @@ checkout_main_if_new() {
 
 build_containers() {
   echo "Yep, definitely building containers. That's what this function does" >&2
+  docker-build-hub apps/crawler/crawler/http/Dockerfile
```

Comment:
Do we want to build and push to docker hub or only build to local?  This seems like it's going to generate a lot of transit data. Moreover, given the way you're doing the testing, if we want to test on arm it would be a separate build/test run and then the two builds would clash.

Reply:
Personally, I'd find it convenient that reasonably up-to-date images are easily available to test with.  Having them in DockerHub seems like the lowest-friction way.